locales/ja/wysiwyg.json

```diff
@@ -0,0 +1,326 @@
+    "Find": "検索",
+    "Find and replace": "検索・置換",
+    "Find and replace...": "検索・置換..",
+    "Find whole words only": "全文検索のみ",
+    "Finish": "完了",
+    "Flags": "旗・国旗",
+    "Font": "フォント",
+    "Font Sizes": "フォントサイズ",
+    "Fonts": "フォント",
+    "Food and Drink": "食べ物・飲み物",
+    "Footer": "フッター",
+    "Format": "形式",
+    "Format Painter": "Format Painter",
+    "Formats": "形式",
+    "Fullscreen": "全画面",
+    "G": "G",
+    "General": "基本",
+    "Gray": "グレー",
+    "Green": "緑",
+    "H Align": "見出しの位置",
+    "Header": "見出し",
+    "Header 1": "見出し H1",
+    "Header 2": "見出し H2",
+    "Header 3": "見出し H3",
+    "Header 4": "見出し H4",
+    "Header 5": "見出し H5",
+    "Header 6": "見出し H6",
+    "Header cell": "見出しのセル",
+    "Headers": "見出し",
+    "Heading 1": "見出し H1",
+    "Heading 2": "見出し H2",
+    "Heading 3": "見出し H3",
+    "Heading 4": "見出し H4",
+    "Heading 5": "見出し H5",
+    "Heading 6": "見出し H6",
+    "Headings": "Headings",
+    "Height": "縦幅",
+    "Help": "ヘルプ",
+    "Horizontal line": "水平線",
+    "Horizontal space": "水平スペース",
+    "Id": "ID",
+    "Id should start with a letter, followed only by letters, numbers, dashes, dots, colons or underscores.": "ID は文字で始まり、文字、数字、ダッシュ、ドット、コロン、アンダースコアで構成されている必要があります。",
+    "Ignore": "無視",
+    "Ignore all": "すべて無視",
+    "Image": "画像",
+    "Image description": "画像の説明",
+    "Image list": "画像一覧",
+    "Image options": "画像オプション",
+    "Image title": "画像名",
+    "Image...": "画像...",
+    "Increase indent": "インデントを増やす",
+    "Inline": "インライン",
+    "Insert": "挿入",
+    "Insert column after": "右に 1 桁挿入",
+    "Insert column before": "左に 1 桁挿入",
+    "Insert date/time": "日時の挿入",
+    "Insert image": "画像の挿入",
+    "Insert link": "リンクの挿入",
+    "Insert row after": "下に 1 行導入",
+    "Insert row before": "上に 1 行挿入",
+    "Insert table": "テーブルの挿入",
+    "Insert video": "ビデオの挿入",
+    "Insert/Edit Link": "リンクの追加・編集",
+    "Insert/edit iframe": "Iframe の挿入・編集",
+    "Insert/edit image": "画像の追加・編集",
+    "Insert/edit link": "リンクの挿入・編集",
+    "Insert/edit media": "Media の挿入・編集",
+    "Insert/edit video": "Video の挿入・編集",
+    "Italic": "斜体",
+    "Justify": "Justify",
+    "Keyboard Navigation": "キーボードナビゲーション",
+    "Language": "言語",
+    "Left": "左",
+    "Left to right": "左 → 右",
+    "Light Blue": "水色",
+    "Light Gray": "薄いグレー",
+    "Light Green": "黄緑",
+    "Light Purple": "薄い紫",
+    "Light Red": "薄い赤",
+    "Light Yellow": "薄い黄色",
+    "Link": "リンク",
+    "Link list": "リンク一覧",
+    "Link...": "リンク...",
+    "Loading emoticons...": "絵文字を読込中...",
+    "Lower Alpha": "英小文字",
+    "Lower Greek": "ギリシャ小文字",
+    "Lower Roman": "ローマ数字小文字",
+    "Match case": "Match case",
+    "Mathematical": "Mathematical",
+    "Media": "Media",
+    "Media poster (Image URL)": "Media poster (画像 URL)",
+    "Media...": "Media...",
+    "Medium Blue": "Medium Blue",
+    "Medium Gray": "薄めのグレー",
+    "Medium Purple": "Medium Purple",
+    "Merge cells": "セルで統合",
+    "Middle": "Middle",
```

Comment:
```suggestion
    "Middle": "真ん中",
```

Reply:
What is this used for? The meaning can be strange.

Reply:
I believe is in respect to the position of the element. "Middle" and "Center" are kind of synonyms.